addons/account_bank_statement_import_ofx/demo/demo_data.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
and that's probably because of the behavior change discussed above that you had to add this demo data, but that's not needed if you fix